phpstan.dist.neon

```diff
@@ -1,10 +1,12 @@
 parameters:
-    level: 8
+    level: 10
     paths:
         - src
         - tests
-    checkMissingIterableValueType: false
```

Comment:
note: This option has been removed, plus we probably don't really want it anyway.